app/(pages)/ecommerce/_components/EcommercePricingPlansSection/CardTab.tsx

```diff
@@ -0,0 +1,34 @@
+import { FunctionComponent } from "react";
+import { useFormContext } from "react-hook-form";
+
+import { Button } from "components";
+import { cn } from "utils/cn";
+
+import { IPricingPlanForm } from "../../_interfaces/IPricingPlanForm";
+
+type TProps = {
+  text: string;
+};
+
+export const CardTab: FunctionComponent<TProps> = ({ text }) => {
+  const { watch, setValue } = useFormContext<IPricingPlanForm>();
+  const activePlan = watch("plan");
+  const isActive = text === activePlan;
+  const handleClick = () => {
+    setValue("plan", text);
+  };
+
+  return (
+    <Button
+      onClick={handleClick}
+      className={cn(
+        "md:text-base text-sm py-4 px-5 rounded-t-xl w-full flex justify-center items-center duration-300",
+        {
+          "bg-accent": isActive,
```

Comment:
```suggestion
          "bg-accent": text === activePlan,
```
just use it in place, as we don't have other usage for the `isActive` variable